components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java

```diff
@@ -472,6 +473,39 @@ public void removeWorkflow(String workflowId) throws WorkflowException {
         }
     }
 
+    /**
+     * Remove all workflows by tenant id.
+     *
+     * @param tenantId  Id of the tenant
+     * @throws WorkflowException
+     */
+    @Override
+    public void removeWorkflows(int tenantId) throws WorkflowException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Deleting all workflows of tenant: " + tenantId);
+        }
+
+        List<WorkflowListener> workflowListenerList = WorkflowServiceDataHolder.getInstance().getWorkflowListenerList();
+
+        // Invoke onPreDelete on workflow listeners
```

Comment:
The comments should start with a capital and end with a full stop. Fix all the other comments accordingly.

Reply:
Fixed in #3312